dips.xamarin.ui.nuspec

```diff
@@ -1,33 +1,37 @@
 <?xml version="1.0"?>
 <package >
     <metadata>
-    <id>DIPS.Xamarin.UI</id>
-    <version>1.0.0</version>
-    <title>DIPS.Xamarin.UI</title>
-    <authors>DIPS AS</authors>
-    <owners>DIPS AS</owners>
-    <iconUrl>https://raw.githubusercontent.com/DIPSAS/DIPS.Xamarin.UI/master/assets/DIPS_Xamarin_UI_128x128.png</iconUrl>
-    <tags>xamarin forms xamarinforms xamarin.forms xamarinui xaml</tags>
-    <license type="expression">MIT</license>
-    <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <copyright>© DIPS AS. All rights reserved.</copyright>
-    <description>A shared UI library that DIPS uses and maintains for their mobile applications.</description>
-    <dependencies>
-        <group targetFramework="netstandard2.0">
-            <dependency id="Xamarin.Essentials" version="1.3.0" />
-            <dependency id="Xamarin.Forms" version="4.7.0" />
-        </group>
-        <group targetFramework="MonoAndroid">
-            <dependency id="Xamarin.Essentials" version="1.3.0" />
-            <dependency id="Xamarin.Forms" version="4.7.0"/>
-        </group>
-        <group targetFramework="Xamarin.iOS10">
-            <dependency id="Xamarin.Essentials" version="1.3.0" />
-            <dependency id="Xamarin.Forms" version="4.7.0" />
-        </group>
-    </dependencies>
+        <id>DIPS.Xamarin.UI</id>
```

Comment:
Theres a lot of changes here, but I've just tabbed in the content and added:
- More tags
- Project URL
- Licence points to a file
- Added Licence file
- Added 2020 to the copyright